From b39fd79099df743495cdded2ec8b27ff94983567 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jos=C3=A9=20Dapena=20Paz?= <jose.dapena@lge.com>
Date: Wed, 8 Nov 2017 17:52:15 +0100
Subject: Fix GCC build of angle

size_t was not defined, so Constants.h inclussion would make
compilation fail.

diff --git a/src/common/platform.h b/src/common/platform.h
index bc711f8f1..74d6859b5 100644
--- a/src/common/platform.h
+++ b/src/common/platform.h
@@ -92,7 +92,7 @@
 #endif
 
 // Mips devices need to include stddef for size_t.
-#if defined(__mips__)
+#if defined(__mips__) || defined(__GNUC__)
 #include <stddef.h>
 #endif
 
-- 
2.14.1

